src/overlays/actors/ovl_En_Ru1/z_en_ru1.c

```diff
@@ -90,6 +90,19 @@ void EnRu1_DrawNothing(EnRu1* this, PlayState* play);
 void EnRu1_DrawOpa(EnRu1* this, PlayState* play);
 void EnRu1_DrawXlu(EnRu1* this, PlayState* play);
 
+typedef enum EnRu1Type {
+    ENRU1_TYPE_BOSS_ROOM,
+    ENRU1_TYPE_FOUNTAIN,
+    ENRU1_TYPE_HOLES_ROOM,
+    ENRU1_TYPE_BASEMENT,
+    ENRU1_TYPE_SAPPHIRE_ROOM,
+    ENRU1_TYPE_BESIDE_KZ,
+    ENRU1_TYPE_BESIDE_DOOR_SWITCH,
+#if DEBUG_FEATURES
+    ENRU1_TYPE_DEBUG = 10,
+#endif
+} EnRu1Type;
```

Comment:
I should have been more clear sorry, the enum should be in the header as well.
Its common that actors include other actors and check their params to gain information about them